File: CodeHelper/CodeHelper.Core/Parse/ParseResults/Workflows/Clz.cs
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

using CodeHelper.Core.Parse.ParseResults;
using System;
using CodeHelper.Core.Parse.ParseResults.Workflows;
using System.Collections.Generic;
namespace CodeHelper.Core.Parse.ParseResults.Workflows
{
    public class Clz : TokenPair
    {
        public String Name { get; set; }
        public List<Field> Fields { get; set; }


        internal ClzInfo Parse()
        {
            var rslt = new ClzInfo();
            rslt.TokenPair = this;
            rslt.Name = Name;

            foreach (var f in this.Fields)
            {
                rslt.fields.Add(f.Parse());
            }

            return rslt;
        }
    }
}

File: ToolBag/ParseTool/src/parsetool/workflow/models/Clz.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package parsetool.workflow.models;

import java.util.ArrayList;
import java.util.List;
import parsetool.models.common.TokenPair;

/**
 *
 * @author Administrator
 */
public class Clz extends TokenPair{
    private String name;
    private List<Field> fields = new ArrayList<Field>();

    /**
     * @return the name
     */
    public String getName() {
        return name;
    }

    /**
     * @param name the name to set
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * @return the fields
     */
    public List<Field> getFields() {
        return fields;
    }

    /**
     * @param fields the fields to set
     */
    public void setFields(List<Field> fields) {
        this.fields = fields;
    }
}
 